x-pack/test_serverless/api_integration/services/svl_cases/api.ts

```diff
@@ -14,17 +14,17 @@ import { CasesFindResponse } from '@kbn/cases-plugin/common/types/api';
 import { kbnTestConfig, kibanaTestSuperuserServerless } from '@kbn/test';
 import { FtrProviderContext } from '../../ftr_provider_context';
 
+export interface User {
+  username: string;
+  password: string;
+  description?: string;
+  roles: string[];
+}
```

Comment:
I can't see that `User` is used outside of this file?

Reply:
I'm still verifying these, but I these kinds of "export" additions were a kind of typescript errors that were something like an inferred type was private - these additions fixed those errors, but now that I'm re-verifying, both User and CommonSavedObjectAttributes works without export, so I'm puzzled 😕

Reply:
IDE TypeScript errors work in mysterious ways 🤷

Reply:
it wasn't an IDE, it was through the typecheck command. But it no longer seems to be present, so we can undo these, or keep them